batcher-bot/liquidity/src/index.ts

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+import { TezosToolkit } from "@taquito/taquito";
+import { InMemorySigner } from "@taquito/signer";
+import { run_jit, run_always_on } from "./bot";
+import { contract_details, liquidity_settings } from "./types";
+import { load_settings } from "./settings";
+import { HubConnection, HubConnectionBuilder } from "@microsoft/signalr";
+import { get_contract_detail_from_storage, echo_terminal } from "./utils";
+import { Option } from "prelude-ts";
+const clear = require("clear");
+const { Command } = require("commander");
+const cli = new Command();
+
+clear();
+
+echo_terminal("Batcher Bot", Option.none());
+
+const get_connection = (uri: string): HubConnection => {
+  return new HubConnectionBuilder().withUrl(uri + "/v1/ws").build();
+};
+const preload = async (
+  tezos: TezosToolkit,
+  settings: liquidity_settings
+): Promise<contract_details> => {
+  const contract_uri: string = `${settings.tzkt_uri_api}/v1/contracts/${settings.batcher_address}/storage`;
+  console.info("contract_uri", contract_uri);
+  tezos.setProvider({
+    signer: await InMemorySigner.fromSecretKey(
+      process.env["TEZOS_PRIV_KEY"] || "No priv key defined"
```

Comment:
Should we instead just throw here since the bot won't work without the key anyway?

Reply:
Agreed.  Will change